hpke.go

```diff
@@ -106,13 +159,13 @@ func verifyMode(suite CipherSuite, mode HPKEMode, psk, pskID, pkIm []byte) error
 
 	ok := false
 	switch mode {
-	case modeBase:
+	case ModeBase:
```

Comment:
If we keep the tests inside the module boundary, no need for these constants to be public.